ansible/roles/atmo-fail2ban/vars/main.yml

```diff
@@ -1,6 +1,9 @@
 SSH_PORT: '22'
 BANTIME: '1800'
-MAX_RETRY: '3'
```

Comment:
Let's set these in group_vars per-environment and only leave reasonable defaults here?